apps/tangle-dapp/containers/ValidatorTablesContainer/ValidatorTablesContainer.tsx

```diff
@@ -51,7 +53,7 @@ const ValidatorTablesContainer = () => {
             "
             buttonText="Learn More"
             buttonProps={{
-              onClick: () => window.open(TANGLE_STAKING_URL, '_blank'),
```

Comment:
We can use href and target here to improve accessibility. The `Button` component already handles rendering either `<button />` or `<a />` based on the props.